README.md

```diff
@@ -11,6 +11,17 @@ The `dbt-database-adapter-scaffold` package is a user-friendly interactive way t
 
 ## Getting started
 
+### Local environment setup
+- cd to where you'd like your adapter to live
+- setup a virtual env
+```
+python3 -m venv env
+```
+- choose how you would like to run the `dbt-database-adapter-scaffold`
+      - Running from Github
+      - Running Locally
+- Once you have generated your adapter please continue by using the `README` located within the repo.
```

Comment:
Do we have a recommendation? I'm guessing that running from GitHub is simpler/faster?
```suggestion
Choose how you would like to run the scaffold:
- Running from Github (recommended)
- Running locally

Once you have generated your adapter, please continue by using the `README` located within the repo.
```

Reply:
We hadn't picked one specifically yet was also curious if we were going to want to make this into a pypi package or any other form of possible download?